### PR TITLE
Remove options from someone-else journey

### PR DESCRIPTION
**Pull request by easternbloc**
Previously you could nominate, elect or cancel someone-else to collect your brp.

The new form just allows you to nominate someone. If you need to change that person you just nominate again.

Jira BRP-169

### CODE REVIEW

File: apps/someone-else/controllers/arrange.js [hunk 1]
```diff
@@ -32,24 +32,7 @@ ArrangeController.prototype.processDate = function processDate(key, values) {
...
-  if (req.form.values['arrange-collection-radio'] === 'change-person') {
-    req.form.values['no-reason'] = true;
-    req.form.values.nominating = 'Change';
-    this.options.next = '/personal-details-no-reason';
-  }
-
-  if (req.form.values['arrange-collection-radio'] === 'cancel-request') {
-    this.options.next = '/exit-cancel-request';
-  }
-
```
Comment (daniel-ac-martin) [inline apps/someone-else/controllers/arrange.js]: FYI, this change means that this controller would be unnecessary if and when we have date functionality in the HOF buttery biscuit controller.

File: apps/someone-else/views/arrange.html [hunk 2]
```diff
@@ -26,14 +26,11 @@
...
   {{$content}}
-    <p>{{#t}}pages.arrange-collection.subheader{{/t}}</p>
-
     {{<partials-form}}
 
       {{$form}}
-        {{#radio-group}}arrange-collection-radio{{/radio-group}}
 
-        <div id="someone-else-group" class="panel-indent">
+        <div id="someone-else-group">
```
Comment (daniel-ac-martin) [inline apps/someone-else/views/arrange.html]: Do we even need this tag now?

File: apps/someone-else/views/confirm.html [hunk 3]
```diff
@@ -89,49 +89,6 @@ <h2 class="details-header">{{#t}}pages.someone-else-check-details.tables.other-p
         </table>
       {{/values.someone-else-fullname}}
```
Comment (daniel-ac-martin) [inline apps/someone-else/views/confirm.html]: Isn't this always there now? Could we get rid of the conditional?

File: apps/someone-else/controllers/arrange.js (hunk 1 above)
Comment (easternbloc) [inline apps/someone-else/controllers/arrange.js]: Sadly when :(

Comment (easternbloc) [inline apps/someone-else/controllers/arrange.js]: I wanted the buttery biscuit controller but instead I have just a dry water biscuit

Comment (daniel-ac-martin) [inline apps/someone-else/controllers/arrange.js]: Sadly if. :-(

**Comment by easternbloc** [comment]
@daniel-ac-martin force pushed...

**Pull request merged**